bookshelf.js

```diff
@@ -39,4 +40,16 @@ spines.map(function (s, i) {
   covers[i].style.top = `${280 - randomHeight}px`;
 
   tops[i].style.top = `${280 - randomHeight}px`;
+
+  // 75 characters above spoils the alignment
+  if (titles[i].innerHTML.length > 75) {
+    titles[i].innerHTML = titles[i].innerHTML.substring(0, 73) + "...";
+  }
+
+  //If all caps capitalize the first letter of each word as all capitals also spoil the alignment
+  if (titles[i].innerHTML.toUpperCase() === titles[i].innerHTML) {
+    titles[i].innerHTML = titles[i].innerHTML.replace(/\w\S*/g, function (txt) {
+      return txt.charAt(0).toUpperCase() + txt.substr(1).toLowerCase();
+    });
+  }
```

Comment:
Can `text-transform: capitalize;` not be used instead?

Reply:
Here I am using this to only capitalize the first letter of the already capitalized string

For example - HELLO WORLD gets converted to Hello World

Reply:
Right, I don't think you need all that JS for that though. 

`text-transform: capitalize` will take lower case text and turn it into "title case" (`hello world -> Hello World`) so to make it work with all caps input you can just use `toLowerCase()`

I'm just wondering if this is really necessary though; the user defines the input after all. Need to think about it.

Reply:
Actually yes this makes sense, I totally skipped the part where is the user is the one who enters the data. Should I put in a commit removing this piece of code?

Reply:
Yes, I think ultimately this isn't needed. However, adding: 
```css
    text-transform: capitalize;
    text-align: left;
```
to `.spine-title` is a good addition I think.

Reply:
For some reason `text-align: left;` makes it look bad. The center alignment should stay I feel.